zerver/views/users.py

```diff
@@ -452,7 +453,7 @@ def add_bot_backend(
     bot_type: Json[int] = UserProfile.DEFAULT_BOT,
     payload_url: Json[Annotated[str, AfterValidator(check_url)]] = "",
     service_name: str | None = None,
-    integration_name: Optional[str] = None,
```

Comment:
Please fix new changes in the same commit it is introduced, and not in the following commit.

Reply:
Changes made